yt/yt/ytlib/api/native/config.h

```diff
@@ -360,6 +360,9 @@ struct TConnectionDynamicConfig
 
     THashSet<std::string> BannedInSyncReplicaClusters;
 
+    //! Enables strict access validation in scheduler commands.
+    bool StrictSchedulerCommandsAccessValidation;
```

Comment:
"Scheduler commands" is too broad. I would suggest `StrictOperationInfoAccessValidation` or something like that.

Reply:
I chose it because of [scheduler_commands.h](https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/client/driver/scheduler_commands.h)/.cpp files and some mentions of "scheduler" commands

`get_job` doesn't seem to belong to `OperationInfo` commands, but yt config [already has](https://github.com/ytsaurus/ytsaurus/blob/5f3a4f4dedb5583b856f3ced3b2a97f568422601/yt/python/yt/wrapper/default_config.py#L666) `operation_info_commands_timeout` which includes get_job so I will change it to `StrictOperationInfoAccessValidation`

Reply:
Scheduler commands includes all kind of things, e.g. starting an operation, aborting a job, etc.

Regarding `get_job` and `list_jobs`, one can argue that a job is a part of an operation, so they also could fall under the "operation info" label.